src/App.tsx

```diff
@@ -1,7 +1,20 @@
 import { OnboardingPage } from './pages/OnboardingPage';
+import { IntlProvider } from 'react-intl';
+import { useState } from 'react';
+import { languages, locales } from './i18n/languages.ts';
+import LocaleSwitcher from '@components/LocaleSwitcher.tsx';
 
 function App() {
-    return <OnboardingPage />;
+    const defaultLocale = navigator.language ? navigator.language : locales.de;
```

Comment:
You can simplify it with [Nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)

```suggestion
    const defaultLocale = navigator.language ?? locales.de;
```

Reply:
Thanks for the input :)